compiler/sem/semmagic.nim

```diff
@@ -401,6 +401,15 @@ proc magicsAfterOverloadResolution(c: PContext, n: PNode,
       result = semAsgnOpr(c, n)
     else:
       result = semShallowCopy(c, n, flags)
+  of mEnumToStr:
+    # overload resolution picked the generic enum-to-string magic.
+    # Replace the symbol with that of the auto-generated procedure.
+    let
+      info = n[0].info
+      prc = getToStringProc(c.graph, n[1].typ.skipTypes(abstractRange))
+    result = n
+    result[0] = newSymNode(prc)
+    result[0].info = info
```

Comment:
```suggestion
    result[0] = newSymNode(prc, info)
```

Soft suggesiotn, but it saves a a line. :D

Reply:
D'oh, I forgot about this overload existing. Thanks for the reminder :D